<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProveedoresForm</class>
 <widget class="QDialog" name="frmProveedoresForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources.qrc">
    <normaloff>:/assets/proveedores.png</normaloff>:/assets/proveedores.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Página web:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtTelefono"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtNombre"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Dirección:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nombre del proveedor:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAceptar">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QPlainTextEdit" name="txtHorarios">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="txtDireccion">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Horarios:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Observaciones:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Teléfono:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="txtPaginaWeb"/>
   </item>
   <item row="6" column="1">
    <widget class="QPlainTextEdit" name="txtObservaciones">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="txtEmail"/>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="chkMostrarEnInformeVentas">
     <property name="text">
      <string>Mostrar ventas de artículos de este proveedor en informe de ventas.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtNombre</tabstop>
  <tabstop>txtTelefono</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>txtPaginaWeb</tabstop>
  <tabstop>txtDireccion</tabstop>
  <tabstop>txtHorarios</tabstop>
  <tabstop>txtObservaciones</tabstop>
  <tabstop>chkMostrarEnInformeVentas</tabstop>
  <tabstop>btnAceptar</tabstop>
  <tabstop>btnCancelar</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>frmProveedoresForm</sender>
   <signal>rejected()</signal>
   <receiver>frmProveedoresForm</receiver>
   <slot>closeParent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>frmProveedoresForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAceptar</sender>
   <signal>clicked()</signal>
   <receiver>frmProveedoresForm</receiver>
   <slot>SaveProveedor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>closeParent()</slot>
  <slot>SaveProveedor()</slot>
 </slots>
</ui>
